### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_New deployment - Azure Database for MySQL_Screen Reader:NVDA is not announcing  "accept terms of use" label in azure database for my sql dialog

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio-mysql#183]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #GH-MySQLextensionforAzureDataStudio-Win32-Mar23; #MySQL extension for Azure Data Studio; #Win32; #WCAG1.3.1; #Info and Relationships; #NVDA;  #Screen Reader; #Benchmark; #JAWS; #Element:EditField;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
NVDA Version 2023.1
JAWS version 2023
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Tab till new deployment and press enter and Turn on nVDA
2. Tab till azure database for sql and press enter
3. Tab till accept terms of use 
4. Verify Narrator/NVDA is announcing label or not
### Actual Result:
NVDA is not announcing  "accept terms of use" label in azure database for my sql dialog
**Issue repro with JAWS also**

### Expected Result:
NVDA should announce "accept terms of use" label in azure database for my sql dialog
### Notes:
**NVDA attachment:**
![A11y_MySQLextensionforAzureDataStudio_Create new server group_Screen Reader_NOTES_NVDA](https://user-images.githubusercontent.com/100189407/228855398-c6dc89a8-7c7a-4ccf-83df-11b225d31cce.png)
![Screenshot (75)](https://user-images.githubusercontent.com/100189407/229689000-8e3326cf-7fd1-4fdb-ba99-43de0caf4a7a.png)

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by VijayaBharathyA** [comment]
As per comments mentioned https://github.com/microsoft/azuredatastudio/issues/18592 in this by design bug. Screen reader will read in Scan mode hence closing bug.
